### ISSUE TITLE
Karta produktu neexistuje pro CZ a SK trh (EAN 16809491093)

### ISSUE THREAD

**Issue by Servatechsro** [allegro/allegro-api#10257]
Dobrý den,

Bohužel se nám nedaří napojit produkt na kartu 16809491093. Listujeme standardně skrze Baselinker, a s tímto problémem jsme se ještě nesetkali. Na PL Allegru produkt máme již vylistovaný. Nelze nám ho listovat jen na CZ a SK Allegro. Chybová hláška zde:

Pro daný jazyk nabídky nebyl produkt nalezen. Zkuste to znovu později. (ProductLanguageVersionUnavailableException productSet[0].product)

Je zajímavé, že jiná barevná varianta tohoto produktu nám přitom šla vylistovat bez problému. Prosíme tedy o prověření tohoto problému z Vaší strany a zpětnou vazbu, jak máme dále postupovat.

S přáním pěkného dne

Zákaznický tým
Servatech s.r.o. 

![image](https://github.com/user-attachments/assets/cea57cbe-97df-4f6b-86e4-b8146780a964)

**Comment by MartaNowaczyk** [comment]
The product on offer is successfully listed in other marketplaces. Please provide a trace-id for this error case for verification. Such technical data should be available in Baselinker.

**Comment by Servatechsro** [comment]
Hello,

Thank you for your message. Is this the trace-id?

ProductLanguageVersionUnavailableException productSet[0].product

If not, please telll us, where we should find it.

Regards

Customer Team
Servatech s.r.o.

**Comment by MartaNowaczyk** [comment]
No, traceId is a string such as ‘600fe94fc72f40fa’ for example. If Baselinker returns an error file then it should be there. If you can't find it ask Baselinker please.
